### PR TITLE
GEN-1177 - feat(TrialExtensionForm): sign new/existing members

### PR DESCRIPTION
**Pull request by guilhermespopolin**
## Describe your changes

- Car dealership: implement sign for new/existing members

Co-authored: @robinandeer

### CODE REVIEW

**Comment by guilhermespopolin** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3253** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3253" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3253?utm_source=stack-comment).

File: apps/store/src/features/carDealership/TrialExtensionForm.stories.tsx [hunk 1]
```diff
@@ -14,12 +16,17 @@ export const WithoutExtension: Story = {
...
     priceIntent: CAR_TRIAL_DATA_QUERY.priceIntent,
+    shopSession: CAR_TRIAL_DATA_QUERY.shopSession,
   },
 
   render(args) {
     return (
       <AppErrorProvider>
-        <TrialExtensionForm {...args} />
+        <BankIdContextProvider>
+          <ShopSessionProvider>
```
Comment (robinandeer) [inline apps/store/src/features/carDealership/TrialExtensionForm.stories.tsx:26]: why do we need this again? we are not relying on the shop session from the context...

**Review by robinandeer — APPROVED** [review]
Added via Giphy

File: apps/store/src/features/carDealership/TrialExtensionForm.stories.tsx (hunk 1 above)
Comment (guilhermespopolin) [reply apps/store/src/features/carDealership/TrialExtensionForm.stories.tsx:26]: Not directly but `useBankIdContext` uses it

**Pull request merged**